src/Compilers/CSharp/Portable/Syntax/SyntaxKind.cs

```diff
@@ -13,245 +13,462 @@ public enum SyntaxKind : ushort
         RegionKeyword = 8469,
+        /// <summary>Represents <see langword="endregion"/>.</summary>
         EndRegionKeyword = 8470,
+        /// <summary>Represents <see langword="define"/>.</summary>
         DefineKeyword = 8471,
+        /// <summary>Represents <see langword="undef"/>.</summary>
         UndefKeyword = 8472,
+        /// <summary>Represents <see langword="warning"/>.</summary>
         WarningKeyword = 8473,
+        /// <summary>Represents <see langword="error"/>.</summary>
         ErrorKeyword = 8474,
+        /// <summary>Represents <see langword="line"/>.</summary>
         LineKeyword = 8475,
+        /// <summary>Represents <see langword="pragma"/>.</summary>
         PragmaKeyword = 8476,
+        /// <summary>Represents <see langword="hidden"/>.</summary>
         HiddenKeyword = 8477,
+        /// <summary>Represents <see langword="checksum"/>.</summary>
         ChecksumKeyword = 8478,
+        /// <summary>Represents <see langword="disable"/>.</summary>
         DisableKeyword = 8479,
+        /// <summary>Represents <see langword="restore"/>.</summary>
         RestoreKeyword = 8480,
+        /// <summary>Represents <see langword="r"/>.</summary>
         ReferenceKeyword = 8481,
 
+        /// <summary>Represents <c>$"</c> token.</summary>
         InterpolatedStringStartToken = 8482,            // $"
+        /// <summary>Represents <c>"</c> token that is closing <c>$"</c>.</summary>
         InterpolatedStringEndToken = 8483,              // "
+        /// <summary>Represents <c>$@</c> or <c>@$</c> token.</summary>
         InterpolatedVerbatimStringStartToken = 8484,    // $@" or @$"
 
         // additional preprocessor keywords (continued)
+        /// <summary>Represents <see langword="load"/>.</summary>
         LoadKeyword = 8485,
+        /// <summary>Represents <see langword="nullable"/>.</summary>
         NullableKeyword = 8486,
+        /// <summary>Represents <see langword="enable"/>.</summary>
         EnableKeyword = 8487,
 
         // targets for #nullable directive
+        /// <summary>Represents <see langword="warnings"/>.</summary>
         WarningsKeyword = 8488,
+        /// <summary>Represents <see langword="annotations"/>.</summary>
         AnnotationsKeyword = 8489,
 
         // Other
+        /// <summary>Represents <see langword="var"/>.</summary>
         VarKeyword = 8490,
+        /// <summary>Represents <c>_</c> token.</summary>
         UnderscoreToken = 8491,
+        /// <summary>Represents that nothing was specified as a type argument. For example <c>Dictionary&lt;,&gt;</c> which has OmittedTypeArgumentToken as a child of OmittedTypeArgumentSyntax before and after the CommaToken.</summary>
         OmittedTypeArgumentToken = 8492,
+        /// <summary>Represents that nothing was specified as an array size. For example <c>int[,]</c> which has OmittedArraySizeExpressionToken as a child of OmittedArraySizeExpressionSyntax before and after the CommaToken.</summary>
```

Comment:
```suggestion
        /// <summary>Represents that nothing was specified as an array size.</summary>
        /// <remarks>For example <c>int[,]</c> which has <see cref="OmittedArraySizeExpressionToken"/> as a child of <see cref="OmittedArraySizeExpressionSyntax"/> before and after the <see cref="CommaToken"/>.</remarks>
```

Reply:
@sharwell, it was noted by @svick that remarks for enum members doesn't appear in the doc site.

Reply:
We can address this either in the tool that converts our build output to https://github.com/dotnet/roslyn-api-docs/blob/live/dotnet/xml/Microsoft.CodeAnalysis.CSharp/SyntaxKind.xml, or in the site generation itself.

Reply:
> We can address this either in the tool that converts our build output to https://github.com/dotnet/roslyn-api-docs/blob/live/dotnet/xml/Microsoft.CodeAnalysis.CSharp/SyntaxKind.xml

@sharwell Is that docfx? or you're using another tool?